detect_secrets/plugins/high_entropy_strings.py

```diff
@@ -162,6 +169,11 @@ def __init__(self, limit: float = 4.5) -> None:
             limit=limit,
         )
 
+    def validate_result(self, result: str) -> bool:
+        # a base64 string is a multiple of 4 chars
+        result_division = len(result) / 4
+        return bool(result_division == int(result_division))
```

Comment:
I think you can use here `len(result) % 4 == 0` which will be simpler

Reply:
I think you are right 😄